src/webapp/components/ClusterMap.js

```diff
@@ -85,12 +96,18 @@ const ClusterMap = ({ geoJsonData, containerStyle = {} }) => {
   // there is a requestCode but the request object does not exist
   const paramRequestNotFound = requestCode && !paramRequest;
 
+  const noRequestsFound = geoJsonData.length === 0;
```

Comment:
This should check for `geoJsonData.features.length`. `geoJsonData` is the GeoJSON FeatureCollection object, which has a property `features` array that contains the markers. 

For reference, this is what gets returned from the /delivery-needed/requests endpoint: https://crownheightsma.herokuapp.com/api/delivery-needed/requests.json